Updated example notebooks

clarasb opened the pull request:
General updates for Notebook 01 to 08 include:

- env information
- corrected typos
- updated URL's

- additional description in 03

gunbra32 reviewed (approved):
all fine, I guess. Reviewing JNBs is almost impossible as every execution generates numerous changes.

clarasb commented:
General updates for Notebook 03, 05, 06, 09, 10 and 11 include:

- updated env information
- corrected typos
- updated cci store (cciodp to ccizarr)
- updated use cases
- new use cases (Notebooks 05, 06, 09)

TejasMorbagal commented:
Thanks @clarasb for the work here :-)
